Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -291,8 +291,11 @@
 
     private static boolean isLogging_ = false;
 
-    private static boolean checkInstallReferrer_ = false;
-    private static long PLAYSTORE_REFERRAL_FETCH_WAIT_FOR = 5000;
+    // Branch will wait for 1.5 sec to capture the install referrer inorder to provide
+    // accurate attribution . This will be only on the first time user open the app.
+    // @see to skip this please call {@link #disablePlayStoreReferrer() }
+    private static boolean checkInstallReferrer_ = true;
```

Comment:
This is only for Install  and that happens only one time. Subsequent open will not have  this delay. We have option to reset it for any one who is worried  about this deal over accurate tracking.